.github/wait-for-port.sh

```diff
@@ -4,7 +4,7 @@ port=$1
 echo "Waiting for port ${port}"
```

Comment:
improve the message here that it's checking for http 200 on that port

Reply:
Alright, done in 1cdbf6a50b9b2d317db57326ccf9d706b9b98e1f